src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -326,6 +330,92 @@ Copyright (c) .NET Foundation. All rights reserved.
     
   </Target>
 
+  <ItemGroup>
+    <ClsidMap Include="$(IntermediateOutputPath)$(AssemblyName).cslidmap" />
```

Comment:
`.cslidmap`? Is this typo other places as well?

Reply:
It's just here.